docs/Test-Strategy-Info.mdx

```diff
@@ -1,31 +1,167 @@
 import { Meta } from '@storybook/blocks';
+import { Link } from 'react-router-dom';
 
-<Meta title="docs/Test-Strategy-Info" />
+<Meta title="docs/WS-Automation-strategy" />
 
-# Test Strategy for adding integration/cypress tests
+# World Service - Test Automation strategy
 
-This document gives a brief overview on what basis integration and e2e tests are added.
+## Background 
 
-![TestingPyramid](https://user-images.githubusercontent.com/9802855/83626408-42541580-a58d-11ea-9891-30dcd2e5b936.png)
+This document aims to provide high-level guidance on approaching test automation for World Service teams. It specifically seeks to answer the following questions:
 
-## Factors to be considered before deciding on integration or cypress tests
 
-* Test runtime
-* How flaky the test would be (how often the test will fail due to external conditions such as timing out)
-* Realistic environment (does it mock the browser behaviour)
-* Client/Server side rendered (as with JSDom in integration testing, it doesn’t entirely emulate a web browser so client side rendered components have to be tested using cypress)
+- Why should we automate? (Why we write tests)
+- Where should we automate? (What levels)
+- What should we automate? 
+- When should we automate?
+- How should we automate? (Patterns, Anti-Patterns and Practices)
+- Who should automate? (Responsibilities)
```

Comment:
Should we consider linking to parts further down in the document, or perhaps a table of contents?

Reply:
I've added table of contents.